WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -377,18 +377,20 @@ public async Task<Coin> OutpointToCoinAsync(InputRegistrationRequest request, Ca
 		return new Coin(input, txOutResponse.TxOut);
 	}
 
-	public async Task<RoundStateResponse> GetStatusAsync(RoundStateRequest request, CancellationToken cancellationToken)
+	public Task<RoundStateResponse> GetStatusAsync(RoundStateRequest request, CancellationToken cancellationToken)
```

Comment:
It shouldn't be `async`, It was async because there was an async lock.

Reply:
I cannot change it as far as I can see. It would end up refactoring IWabiSabiApiRequestHandler.